gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```

Comment:
This is the same version used by https://github.com/FabricMC/yarn/blob/1.16.5 in case you're wondering why I changed it again.

Reply:
I mean, do we *need* gradle 7? It would be nice to have the latest version but "if it's not broken don't fix it"

Reply:
It doesn't break anything (if it did, I'd fix it before pushing anyway) so who cares?

Reply:
> The backwards compatability aspect doesn't really apply here because this is a branch dedicated to a singular version. If it works, it works.
As long as it works, I guess.